<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>仪表</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>211</width>
          <height>37</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>37</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>37</height>
         </size>
        </property>
        <property name="title">
         <string>GroupBox</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="test">
        <property name="geometry">
         <rect>
          <x>753</x>
          <y>9</y>
          <width>256</width>
          <height>61</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;OpenGL&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QOpenGLWidget" name="openGLWidget">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>83</y>
          <width>366</width>
          <height>501</height>
         </rect>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>150</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>30</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>文件</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit</cstring>
        </property>
       </widget>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="geometry">
         <rect>
          <x>235</x>
          <y>61</y>
          <width>451</width>
          <height>411</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateTimeEdit" name="dateTimeEdit">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>211</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
        </property>
       </widget>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>221</width>
          <height>141</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>3</string>
         </property>
         <property name="foreground">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>252</red>
            <green>233</green>
            <blue>79</blue>
           </color>
          </brush>
         </property>
        </column>
        <item>
         <property name="text">
          <string>1.1</string>
         </property>
         <property name="text">
          <string>2.1</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.2</string>
         </property>
         <property name="text">
          <string>2.2</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <item>
          <property name="text">
           <string>新建子项目</string>
          </property>
          <property name="text">
           <string>2.2.2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1.2.2</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </item>
        </item>
       </widget>
       <widget class="QDial" name="dial">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>120</y>
          <width>100</width>
          <height>100</height>
         </rect>
        </property>
       </widget>
       <widget class="QToolButton" name="toolButton">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>30</y>
          <width>26</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>30</y>
          <width>271</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>200</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>提示</string>
        </property>
       </widget>
       <widget class="QQuickWidget" name="quickWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>410</y>
          <width>181</width>
          <height>161</height>
         </rect>
        </property>
        <property name="resizeMode">
         <enum>QQuickWidget::SizeRootObjectToView</enum>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>510</y>
          <width>118</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QToolButton" name="toolButton_2">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>10</y>
          <width>26</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>0</y>
          <width>166</width>
          <height>25</height>
         </rect>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
       <widget class="QCommandLinkButton" name="commandLinkButton">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>490</y>
          <width>177</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>CommandLinkButton</string>
        </property>
       </widget>
       <widget class="QScrollBar" name="horizontalScrollBar">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>560</y>
          <width>160</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>500</y>
          <width>120</width>
          <height>80</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="page"/>
        <widget class="QWidget" name="page_3"/>
        <widget class="QWidget" name="page_2">
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>30</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>time</string>
          </property>
         </widget>
        </widget>
       </widget>
       <zorder>lcdNumber</zorder>
       <zorder>groupBox</zorder>
       <zorder>label</zorder>
       <zorder>openGLWidget</zorder>
       <zorder>test</zorder>
       <zorder>graphicsView</zorder>
       <zorder>dateTimeEdit</zorder>
       <zorder>treeWidget</zorder>
       <zorder>dial</zorder>
       <zorder>toolButton</zorder>
       <zorder>lineEdit</zorder>
       <zorder>pushButton</zorder>
       <zorder>quickWidget</zorder>
       <zorder>progressBar</zorder>
       <zorder>toolButton_2</zorder>
       <zorder>buttonBox</zorder>
       <zorder>commandLinkButton</zorder>
       <zorder>horizontalScrollBar</zorder>
       <zorder>stackedWidget</zorder>
      </widget>
      <widget class="QWidget" name="tab_3">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;zzzzzz&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;cs&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <attribute name="title">
        <string>页</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>页</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menussss">
    <property name="title">
     <string>ssss</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuccccc">
    <property name="title">
     <string>ccccc</string>
    </property>
    <widget class="QMenu" name="menu111">
     <property name="title">
      <string>111</string>
     </property>
    </widget>
    <addaction name="separator"/>
    <addaction name="menu111"/>
    <addaction name="action222"/>
    <addaction name="action333"/>
   </widget>
   <addaction name="menussss"/>
   <addaction name="menuccccc"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QToolButton" name="toolButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>26</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>ok</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>ok2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>370</y>
       <width>67</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>label</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="action222">
   <property name="text">
    <string>222</string>
   </property>
  </action>
  <action name="action333">
   <property name="text">
    <string>333</string>
   </property>
  </action>
  <action name="action121">
   <property name="text">
    <string>121</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>graphicsView</tabstop>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>treeWidget</tabstop>
  <tabstop>dial</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>quickWidget</tabstop>
  <tabstop>toolButton_2</tabstop>
  <tabstop>commandLinkButton</tabstop>
  <tabstop>toolButton_3</tabstop>
  <tabstop>test</tabstop>
  <tabstop>pushButton_3</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>toolButton_2</sender>
   <signal>clicked()</signal>
   <receiver>test</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>784</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>910</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
